app/views/static/payment.html.erb

```diff
@@ -0,0 +1,92 @@
+<html>
+<meta name="viewport" content="width=device-width, initial-scale=1" />
+<head>
+<%= csrf_meta_tags %>
+<script src="https://js.stripe.com/v3/"></script>
+<script>
+  function stripeTokenHandler(token) {
+    // Insert the token ID into the form so it gets submitted to the server
+    var form = document.getElementById('payment-form');
+    var hiddenInput = document.createElement('input');
+    hiddenInput.setAttribute('type', 'hidden');
+    hiddenInput.setAttribute('name', 'stripeToken');
+    hiddenInput.setAttribute('value', token.id);
+    form.appendChild(hiddenInput);
+
+    // Submit the form
+    fetch('/charge', {
```

Comment:
unfortunately not every client supports `fetch` yet (https://caniuse.com/#feat=fetch). There is a polyfill here https://github.com/github/fetch